src/unix/os390.c

```diff
@@ -530,10 +531,15 @@ int uv_interface_addresses(uv_interface_address_t** addresses, int* count) {
   struct ifreq* ifr;
   struct ifreq* p;
   int count_v6;
+  int r;
 
   /* get the ipv6 addresses first */
   uv_interface_address_t* addresses_v6;
-  uv__interface_addresses_v6(&addresses_v6, &count_v6);
+  r = uv__interface_addresses_v6(&addresses_v6, &count_v6);
+
+  if (r != 0) {
+    return r;
+  }
```

Comment:
I think we may want to continue to attempt to find IPv4 interfaces if `uv__interface_addresses_v6` fails, or at least in some of the failure cases. 

For example, according to https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.2.0/com.ibm.zos.v2r2.bpxbd00/ioctl.htm, `ioctl` with `SIOCGIFCONF6` can fail with:
> **EAFNOSUPPORT**
>    No IPv6 enabled TCP/IP stacks are active.

Reply:
That's a good point. I can make this check more selective, but before we were ignoring *all* errors.